Authentication/SignUp/View/SignupController.swift

```diff
@@ -65,62 +67,120 @@ private extension SignupController {
         bindPasswordElements()
         bindButtons()
         setTextfieldOrder()
+        
+        _viewModel.signUpExecuting.observeNext { [unowned self] executing in
+            if executing {
```

Comment:
@guidomb I don't agree in this one. They both are a way of expressing conditional logic; and their functionality is the same. I find no reason for which you would use `if let` rather than the operator because it implies side effects.

I think that when decision is between two elements, one should try to use the operator. It's more accurate in representing what we want to do, it's clearer to the reader. One do not think about things with the `if/let` mentality; one decides between two options in the same place. 

I agree with you in the length of the function.

I found this phrase that somehow sums up what I tried to say:
```
An if/else statement emphasises the branching first and what's to be done is secondary, while a ternary operator emphasises what's to be done over the selection of the values to do it with.
http://stackoverflow.com/questions/4192225/ternary-operator-vs-if-statement-question-of-prettiness

```